folly/fibers/Fiber.h

```diff
@@ -15,6 +15,7 @@
  */
 #pragma once
 
+#include <complex>
```

Comment:
Sorry. I didn't notice that. 
I just added the header so that it can be build, but now I think it's not a right way to solve the problem.
I couldn't find std::abs<T> in any C++ headers. I can only find std::abs() and std::abs<complex<T>>. I think the author may want to use the former rather than the latter.